keyform/views.py

```diff
@@ -1,10 +1,21 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from django.shortcuts import render
+from django.views.generic.base import TemplateView
+from django.views.generic import CreateView
 from django.views.generic.list import ListView
+from keyform.forms import CreateForm
 from keyform.models import Request
 
-
 class HomeView(ListView):
     model = Request
     template_name = "keyform/home.html"
+
+class NewForm(CreateView):
+	template_name = "keyform/add_form.html"
+	model = Request
+	success_url = "/"
+
+	def get_form_class(self):
+		return CreateForm
```

Comment:
Incorrect indentation. Also spaces instead of tabs.
Also, instead of defining a get_form_class() method, you could just set the `form_class` attribute to `CreateForm`.